android/app/build.gradle

```diff
@@ -213,6 +213,7 @@ if (propFile.canRead()) {
 dependencies {
     // please keep this list sorted
     implementation project(':bugsnag-react-native')
+    compile project(':mapbox-react-native-mapbox-gl')
```

Comment:
Change this to "implementation"

Reply:
Thanks. Also, going to test some markdown behavior in this comment.

Underline under the period<ins>.</ins>
Period next to comma<ins>,</ins>